src/App.js

```diff
@@ -1,23 +1,33 @@
-import logo from './logo.svg';
-import './App.css';
-
+import "./App.css";
+import { useEffect, useState } from "react";
+import ReviewCard from "./components/ReviewCard";
+import Header from "./components/Header";
+import api from "./utils/utils";
 function App() {
+  const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    api.get("/reviews").then((respone) => {
```

Comment:
api.get("/reviews") can be extracted out into your utils.js file as a function that makes the Get reviews request. The goal is to extract the API calls out of the components as much as possible. So here we would import the function from utils and invoke it without needing to pass it a method or a path at all.